babel-esm.config.json

```diff
@@ -0,0 +1,61 @@
+{
+  "presets": [
+    "@babel/preset-typescript",
+    "@babel/preset-react",
+    ["@babel/preset-env",
+      {
+        "loose": true,
+        "modules": false
```

Comment:
this is the key difference between the original babel config and the esm one
the other parts might be unnecessary, but it was easier to stay as close as possible to the original